### PR TITLE
Convert the positioning enums and LayoutView to Proto

### LINKED ISSUE #1723: ProtoConv: Convert layout/positioning

### LINKED ISSUE #1725: ProtoConv: Convert layout/layout_style

### PR DESCRIPTION
**Pull request by timothyfroehlich**
## Tree of downstream PRs as of 2024-11-14

* **PR #1774 (THIS ONE)**:
  `main` ← `wb/froeht/protoconv-layout-positioning`

    * PR #1782:
      `wb/froeht/protoconv-layout-positioning` ← `wb/froeht/1731-meter-data`

Fixes #1725
Fixes #1723

### CODE REVIEW

**Comment by yiqunw700** [comment]
Is this pr ready for review?
Looks like there are some shifts with the screenshots:
1. vertical centered now aligns with bottom
2. horizontal centered now aligns with end

**Comment by timothyfroehlich** [comment]
> Is this pr ready for review? Looks like there are some shifts with the screenshots:
> 
> 1. vertical centered now aligns with bottom
> 2. horizontal centered now aligns with end

It's not, it's still marked as draft.

**Comment by timothyfroehlich** [comment]
@yiqunw700 I can't figure out what's causing the other diffs, do you have any ideas?

File: crates/figma_import/src/figma_schema.rs [hunk 1]
```diff
@@ -716,15 +716,19 @@ impl OverflowDirection {
...
-{
-    fn into(self) -> dc_bundle::legacy_definition::layout::positioning::OverflowDirection {
+impl Into<dc_bundle::definition::layout::OverflowDirection> for OverflowDirection {
+    fn into(self) -> dc_bundle::definition::layout::OverflowDirection {
         match self {
-            OverflowDirection::None => dc_bundle::legacy_definition::layout::positioning::OverflowDirection::None,
-            OverflowDirection::HorizontalScrolling => dc_bundle::legacy_definition::layout::positioning::OverflowDirection::HorizontalScrolling,
-            OverflowDirection::VerticalScrolling => dc_bundle::legacy_definition::layout::positioning::OverflowDirection::VerticalScrolling,
-            OverflowDirection::HorizontalAndVerticalScrolling => dc_bundle::legacy_definition::layout::positioning::OverflowDirection::HorizontalAndVerticalScrolling,
+            OverflowDirection::None => dc_bundle::definition::layout::OverflowDirection::None,
```
Comment (rylin8) [inline crates/figma_import/src/figma_schema.rs:722]: Seems odd that formatting allowed no braces here but braces for the other cases.

**Review by rylin8 — APPROVED** [review]

File: crates/figma_import/src/figma_schema.rs (hunk 1 above)
Comment (timothyfroehlich) [reply crates/figma_import/src/figma_schema.rs:722]: Just line length stuff. None being a lot shorter than HorizontalScrolling

**Pull request merged**